cmd/registry/cmd/root.go

```diff
@@ -43,6 +44,7 @@ func Command(ctx context.Context) *cobra.Command {
 	cmd.AddCommand(resolve.Command(ctx))
 	cmd.AddCommand(delete.Command(ctx))
 	cmd.AddCommand(export.Command(ctx))
+	cmd.AddCommand(exec.Command(ctx))
```

Comment:
super-nit: this is out of alphabetic order

Reply:
Done.